### PR TITLE
[Test Mv] move mkldnn unittests to test dir

### PR DESCRIPTION
**Pull request by jinyouzhi**
### PR types

Others

### PR changes

Others

### Describe

#51853 task28

### CODE REVIEW

**Comment by tianshuo78520a** [comment]
单测需要进行修改
image

**Comment by jinyouzhi** [comment]
@zhiqiu @tianshuo78520a  可以review了，PR-CI-Coverage、PR-CI-Model-benchmark我触发rerun了，目测应该跟pr无关

**Review by tianshuo78520a — DISMISSED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/CMakeLists.txt [hunk 1]
```diff
@@ -852,8 +852,8 @@ if(WIN32)
     DEPS python)
 endif()
 
-if(WITH_MKLDNN)
-  add_subdirectory(mkldnn)
+if(WITH_ASCEND_CL)
```
Comment (zhiqiu) [inline python/paddle/fluid/tests/unittests/CMakeLists.txt]: why change this ?

Comment (jinyouzhi) [reply python/paddle/fluid/tests/unittests/CMakeLists.txt]: sry, made a mistake while solving conflicts.
removed both.

**Comment by jinyouzhi** [comment]
@zhiqiu 改好跑完CI了！

**Review by zhiqiu — APPROVED** [review]
LGTM

**Pull request merged**